cypress.env.json

```diff
@@ -0,0 +1,3 @@
+{
+  "password": "test"
```

Comment:
Perhaps we can find a way to pull the environment variables from the `.env.development file`? Otherwise, we can probably just set the environment variables directly through the test harness (in a different PR).

Reply:
https://github.com/p4ckysm4cky/booru/pull/37 may be helpful for this.